projects/core/src/util/index.ts

```diff
@@ -1,6 +1,7 @@
+export * from './applicable';
 export * from './converter.service';
 export * from './create-from';
 export * from './glob.service';
 export * from './regex-pattern';
+export * from './serialization-utils';
```

Comment:
Please remove this. We don't want to expose this function, as we want to tackle this problem with the global error handling, where we'll re-evaluate this approach.

Reply:
This means that you should copy-paste this function to your lib, for now.

Reply:
I see, didn't understand your recent comment about those utils correctly